### PR TITLE
Multiple unfinished business rework

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

This PR aggregate a selection of glitches:
- PIM-6106: As Peter, I would like to remove the escape parameter not usefull
- PIM-6081: As Julia, when I use a multiselect filter in a grid, I would like to automatically unchecked ALL
- PIM-6058: As Julia, I would like to define as default a view I have not created
- PIM-6090: As Julia, I would like to see my dashboard according to my rights

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | :negative_squared_cross_mark:
| Added Behats                      | :white_check_mark:
| Added integration tests           |:negative_squared_cross_mark:
| Changelog updated                 | :clock1:
| Review and 2 GTM                  | :clock1:
| Micro Demo to the PO (Story only) | :white_check_mark:
| Migration script                  | :negative_squared_cross_mark:
| Tech Doc                          |:negative_squared_cross_mark:

:white_check_mark: Done and pass

:red_circle: Done but fail

:clock1: Done but pending

:negative_squared_cross_mark: Not needed

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Repository/DatagridViewRepositoryInterface.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public function getDatagridViewTypeByUser(UserInterface $user);
      *
      * @deprecated Please use DatagridViewRepositoryInterface::findDatagridViewBySearch instead
      */
-    public function findDatagridViewByUserAndAlias(UserInterface $user, $alias);
+    public function findDatagridViewByAlias($alias);
```
Comment (pierallard) [inline src/Pim/Bundle/DataGridBundle/Repository/DatagridViewRepositoryInterface.php]: BC?

File: src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml [hunk 2]
```diff
@@ -22,6 +45,8 @@ services:
             - '@pim_catalog.repository.completeness'
             - '@pim_catalog.helper.locale'
             - '@pim_user.context.user'
+            - '@pim_catalog.filter.chained'
```
Comment (skeleton) [inline src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml]: bc break

File: src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml [hunk 3]
```diff
@@ -32,5 +57,6 @@ services:
             - '@translator'
             - '@pim_catalog.localization.presenter.datetime'
             - '@security.token_storage'
+            - '@oro_security.security_facade'
```
Comment (skeleton) [inline src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml]: same

File: src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml [hunk 4]
```diff
@@ -13,6 +13,29 @@ services:
     ##
     pim_dashboard.widget.links:
         class: '%pim_dashboard.widget.links.class%'
+        arguments:
```
Comment (skeleton) [inline src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml]: bc break

Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml]: Done

File: src/Pim/Bundle/DataGridBundle/Repository/DatagridViewRepositoryInterface.php (hunk 1 above)
Comment (juliensnz) [inline src/Pim/Bundle/DataGridBundle/Repository/DatagridViewRepositoryInterface.php]: Done

File: src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml (hunk 2 above)
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml]: Done

File: src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml (hunk 3 above)
Comment (juliensnz) [inline src/Pim/Bundle/DashboardBundle/Resources/config/widgets.yml]: Done

File: UPGRADE-1.7.md [hunk 5]
```diff
@@ -19,6 +19,37 @@
...
+                my_custom_export_job_name: pim-job-instance-csv-base-export
+                my_custom_import_job_name: pim-job-instance-csv-base-import
+```
+
+#### You added some fields to your custom job
+
+In this case you will also need to register it in your form provider but aslo declare a custom form. You will find a
+detailed documentation [here](https://docs.akeneo.com/1.7/cookbook/import_export/create-connector.html)
+
+#### You created  afully customized screen for your job
```
Comment (juliensnz) [inline UPGRADE-1.7.md]: fixed

**Pull request merged**